test/libsolidity/syntaxTests/metaTypes/type_expression_nested_type_max_keyword.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    function f(bool b) public pure {
+        type(type(b ? uint : uint).max);
```

Comment:
Is that covering the case I mentioned in https://github.com/ethereum/solidity/pull/14347#discussion_r1245462796? :)
> - `type` (`type(true ? uint : uint).max`).

Then it should be this:
```suggestion
        type(b ? uint : uint).max;
```

Not that the first `type` is not inside the same backticks. It was meant as a description, like in the other bullet points.